### PR TITLE
Cloud Container Compute documentation

### PR DESCRIPTION
**Pull request by enolfc**
# Summary 

Move cloud container compute outside the fedcloud docs

----

**Related issue :**

### CODE REVIEW

**Review by gwarf — APPROVED** [review]
LGTM :+1

File: content/en/users/container/docker.md [hunk 1]
```diff
@@ -26,16 +14,16 @@ ready-to-use Virtual Machine Image with
 
 You can start that image as any other VA available from AppDB:
 
-1.  Go to the [EGI Docker image entry in
-    AppDB](https://appdb.egi.eu/store/vappliance/docker.ubuntu.16.04).
+1. Go to the [EGI Docker image entry in
+   AppDB](https://appdb.egi.eu/store/vappliance/docker.ubuntu.16.04).
```
Comment (andrea-manzi) [inline content/en/users/container/docker.md:18]:
```suggestion
   AppDB](https://appdb.egi.eu/store/vappliance/egi.docker.ubuntu.16.04).
```

File: content/en/users/container/docker.md [hunk 2]
```diff
@@ -1,23 +1,11 @@
...
-    installation
-    instructions](https://docs.docker.com/engine/installation/); or
--   running a container orchestration platform that manages the
-    applications on top of a set of nodes (Kubernetes, Docker Swarm and
-    Mesos are examples of such orchestration platforms).
-
-## EGI Docker VA
-
 The [EGI Docker
 VA](https://appdb.egi.eu/store/vappliance/docker.ubuntu.16.04) is a
```
Comment (andrea-manzi) [inline content/en/users/container/docker.md:10]:
```suggestion
VA](https://appdb.egi.eu/store/vappliance/egi.docker.ubuntu.16.04) is a
```

**Review by andrea-manzi — APPROVED** [review]
LGTM!

**Pull request merged**